<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowLogin</class>
 <widget class="QMainWindow" name="windowLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎使用Codia——登录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>360</height>
    </size>
   </property>
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>294</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>360</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonLoginGoReset">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>120</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>忘记密码</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名/邮箱/手机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLoginUsername">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLoginPassword">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoginGoRegister">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>330</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>没有codia账号？点击这里注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLogin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>120</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarLogin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>progressBarLogin</zorder>
    <zorder>pushButtonLoginGoReset</zorder>
    <zorder>checkBox</zorder>
    <zorder>label</zorder>
    <zorder>lineEditLoginUsername</zorder>
    <zorder>label_2</zorder>
    <zorder>lineEditLoginPassword</zorder>
    <zorder>pushButtonLoginGoRegister</zorder>
    <zorder>pushButtonLogin</zorder>
   </widget>
   <widget class="QFrame" name="frameRegister">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>294</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditRegisterPassword">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRegister">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>120</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRegisterCheckPassword">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>设置密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>绑定邮箱</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRegisterUserphone">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRegisterUsername">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRegisterReturn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>120</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameReset">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>294</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonReset">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>120</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>重置密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>绑定邮箱/手机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认新密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditResetCheckNewPassword">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>验证码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditResetCheckNum">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditResetAccount">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditResetNewPassword">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonResetAcquire">
     <property name="geometry">
      <rect>
       <x>339</x>
       <y>69</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>发送验证码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonResetReturn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>120</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <zorder>frameReset</zorder>
   <zorder>frameRegister</zorder>
   <zorder>frameLogin</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>156</y>
      <width>137</width>
      <height>78</height>
     </rect>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actions">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionsd">
   <property name="text">
    <string>登陆</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>BeginLogin()</slot>
 </slots>
</ui>
